lib/assets/javascripts/cartodb/old_common/header.js

```diff
@@ -475,6 +477,16 @@ cdb.admin.Header = cdb.core.View.extend({
   _onTabClick: function(e) {
     e.preventDefault();
     window.table_router.navigate(this._generateTableUrl(e), {trigger: true});
+  },
+
+
+  _doExport: function(e) {
+    this.mapTab._doExport(e);
```

Comment:
Now that `doExport` and `cancelExport` are being called by an external class, they should be noted as being public methods by removing the underscore from the start of the method name. Also might as well just kill the event here rather than pass it, if it's not doing anything else with the event.